includes/subscriptions/class-wc-payments-subscription-service.php

```diff
@@ -570,6 +571,21 @@ public function prevent_wcpay_subscription_changes( bool $supported, string $fea
 		return in_array( $feature, $this->supports, true ) || isset( $this->feature_support_exceptions[ $subscription->get_id() ][ $feature ] );
 	}
 
+	/**
+	 * Remove pending renewal order creation from admin edit subscriptions page.
+	 *
+	 * @param array $actions Array of available actions.
+	 * @return array Array of updated actions.
+	 */
+	public function prevent_wcpay_manual_renewal( array $actions ) {
+		global $theorder;
+
+		if ( wcs_is_subscription( $theorder ) && self::is_wcpay_subscription( $theorder ) ) {
+			unset( $actions['wcs_create_pending_renewal'] );
```

Comment:
We'll also need to unset the `wcs_create_pending_parent` action here. It's somewhat difficult to get that to show up but you can achieve it by:

1. Creating a subscription manually. 
2. Set the status to active.
3. Add line items and a customer.
4. Using the changes in #3095, 
      - log in as the customer
      - go to the My Account subscription page and set up your subscription with WCPayments via the add payment flow. 
5. View the subscription actions as an admin user. 

The `wcs_create_pending_parent` action would be shown but we wouldn't want admin to create pending parent orders for WCPay Subscriptions either. 

If we're being super cautious we would also want to unset `wcs_process_renewal`. At the moment it is won't be shown for WCPay Subscriptions given they don't support [date changes](https://github.com/automattic/woocommerce-subscriptions-core/blob/trunk/includes/admin/class-wcs-admin-meta-boxes.php#L182), if in the future we do support date changes, we would still not want it to be shown, so might as well remove it.

Reply:
Awsome.